src/imars3d/backend/corrections/ring_removal.py

```diff
@@ -6,13 +6,121 @@
 from imars3d.backend.util.functions import clamp_max_workers
 import scipy
 import numpy as np
+import bm3d_streak_removal as bm3dsr
 from multiprocessing.managers import SharedMemoryManager
 from tqdm.contrib.concurrent import process_map
 from functools import partial
 
 logger = logging.getLogger(__name__)
 
 
+class bm3d_ring_removal(param.ParameterizedFunction):
+    """
+    Remove ring artifact from sinograms using BM3D method.
+
+    ref: `10.1107/S1600577521001910 <http://doi.org/10.1107/S1600577521001910>`_
+
+    Parameters
```

Comment:
Missed a bunch of parameters in the docs

Reply:
thanks for  catching this, I will add them back now